boards/earlgrey-nexysvideo/Makefile

```diff
@@ -36,3 +36,12 @@ flash-app: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).elf
 	$(RISC_PREFIX)-objcopy --update-section .apps=$(APP) $^ $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM)-app.elf
 	$(RISC_PREFIX)-objcopy --output-target=binary $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM)-app.elf $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM)-app.bin
 	$(OPENTITAN_TREE)/build-out/sw/host/spiflash/spiflash $(FLASHID) --input=$(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM)-app.bin
+
+test:
+	mkdir -p $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/deps
+	riscv64-linux-gnu-gcc -ggdb3 -c -mabi=ilp32 -march=rv32imc ../../arch/rv32i/src/asm.S -o $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/asm.o
```

Comment:
it seems like a bit of a shame to require the ability to build the asm just to run these tests, especially since the asm should never really change. Could we just include a prebuilt .o file in boards/earlgrey-nexysvideo somewhere, and have a second target for re-generating that file from the .S file? It would be nice to add these tests to CI, without adding dependencies that can't be installed via rustup.

Reply:
I have added a prebuilt test as well that users can run

Reply:
Ok, it now uses a prebuilt binary and there is a make target to rebuild the binary.